utils/hare_cov/README.md

```diff
@@ -13,13 +13,19 @@
 5.  Run the `hare_coverage.py`. It will generate the `xml` report inside 
     `hare_coverage_report/coverage.xml`.
 
+NOTE: For `C` coverage `hax` binary should be executed. As `hax` binary is not executed
```

Comment:
Not sure that it works like it is stated. `hax` binary is not the point.

The fact is that we have `hax.c` file. Python code interacts with `hax.c` and with `libmotr` via FFI layer. In order to make C coverage work, the execution path should go into FFI layer. That layer is intentionally mocked in all automated tests. In other words, we don't have any test that would depend on real `libmotr`. Perhaps that can be done when we implement system tests (currently we have no such tests).

Reply:
I am referring systemd hare-hax service which will call `/opt/seagate/cortx/hare/bin/hax`.  While doing bootstrap hare-hax service started and thus `hax` and that will create `gcno` for hax.c which will be used for the coverage calculation. I don't have much knowledge about but this is my observation.

Yes, It will work for system tests. In future when we implement system tests then we just need to run that for actual coverage calculation till that this script will create 0 coverage output.